python/ray/experimental/state.py

```diff
@@ -397,9 +419,11 @@ def dump_catapult_trace(self, path):
     Args:
       path: The filepath to dump the profiling information to.
     """
-    task_info = self.task_profiles()
+    if end == None:
```

Comment:
we should be able to pass the arguments directly through to `self.task_profiles` without changing their values here